riptide-core/src/main/java/org/zalando/riptide/Requester.java

```diff
@@ -30,8 +32,14 @@
 
 public abstract class Requester extends Dispatcher {
 
+    private final Multimap<String, String> query = LinkedHashMultimap.create();
     private final HttpHeaders headers = new HttpHeaders();
 
+    public final Requester query(final String name, final String value) {
```

Comment:
Yes, if we change it then to `queryParam`. `param` itself has too many possible meanings in the context of HTTP: query param, post param, form param, ...